config.json

```diff
@@ -716,7 +868,15 @@
         "slug": "strain",
         "name": "Strain",
         "uuid": "d51edc8a-893b-4b05-912f-759e0d85123b",
-        "prerequisites": [],
+        "prerequisites": [
+          "anonymous-functions",
+          "lists",
+          "recursion",
+          "tail-call-recursion",
+          "cond",
+          "case",
+          "if"
+        ],
```

Comment:
This exercise says not to use `Enum` functions. I am not sure if that means list comprehensions are not allowed as well, or the opposite, they should be a prerequisite? My approach so far is that list comprehensions are syntactic sugar on top of `Enum`, so I didn't list them anywhere as a prerequisite.